interfaces/builtin/content.go

```diff
@@ -165,9 +179,15 @@ func mountEntry(plug *interfaces.Plug, slot *interfaces.Slot, relSrc string, ext
 	options := make([]string, 0, len(extraOptions)+1)
 	options = append(options, "bind")
 	options = append(options, extraOptions...)
+	source := resolveSpecialVariable(relSrc, slot.Snap)
+	target := resolveSpecialVariable(plug.Attrs["target"].(string), plug.Snap)
+	if _, ok := slot.Attrs["source"].(map[string]interface{}); ok {
```

Comment:
Maybe a small comment like: `// check if new style syntax is used` ? Or maybe even a tiny helper. AIUI this check for the slot.Attr["source"].(map...) is just doing that, right?